schema/components.json

```diff
@@ -565,6 +565,105 @@
           ]
         }
       ]
+    },
+    "PublicListing": {
+      "type": "object",
+      "title": "Public listing",
+      "description": "Details of a publicly listed company, its securities (shares and other tradable financial instruments related to the entity), and related regulatory filings.",
+      "required": [
+        "securitiesListings",
+        "hasPublicListing"
+      ],
+      "minProperties": 1,
+      "properties": {
+        "hasPublicListing": {
+          "type": "boolean",
+          "title": "Has public listing",
+          "description": "This value MUST be true if the entity is known to be a publicly listed company."
+        },
+        "companyFilingsURLs": {
+          "type": "array",
+          "title": "Company filings URLs",
+          "description": "URL or URLs where regulatory filings related to major holdings can be retrieved. URLs may point to pages maintained by regulatory bodies, stock exchanges or by the company itself.",
+          "items": {
+            "type": "string"
```

Comment:
Should we add '"format": "uri"' here too?

Reply:
yep added